app/presenters/organisations/header_presenter.rb

```diff
@@ -0,0 +1,88 @@
+module Organisations
+  class HeaderPresenter
+    attr_reader :org
+
+    def initialize(organisation)
+      @org = organisation
```

Comment:
Nitpick: `@org` can just be `@organisation`.

Reply:
I'm trying to keep it terse. I was originally using organisation and the code was beginning to look pretty dense.